src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,378 @@
+                        "sgRNA",
+                        "pgRNA"
+                    ]
+                },
+                "tiling_modality": {
+                    "title": "Tiling Modality",
+                    "description": "The tiling modality of guides across elements or regions in a CRISPR library.",
+                    "comment": "Peak tiling = all possible gRNA in a selection of peaks. Full tiling = all possible gRNA across an entire region, including regulatory elements and non regulatory elements. Sparse peaks = a small set of gRNA against a selection of peaks.",
+                    "type": "string",
+                    "enum": [
+                        "peak tiling",
+                        "full tiling",
+                        "sparse peaks"
+                    ]
+                },
+                "average_coverage": {
+                    "title": "Average Coverage",
+                    "description": "The average number of guides targeting each element of interest in the library.",
+                    "comment": "May not be submitted with coverage_range.",
+                    "type": "number",
+                    "minimum": 0
+                },
+                "coverage_range": {
+                    "title": "Coverage Range",
+                    "description": "The numerical range of guides targeting each element of interest in the library.",
+                    "comment": "May not be submitted with average_coverage.",
+                    "type": "string",
+                    "pattern": "(^[0-9]+-[0-9]+$|^[<>][0-9]+$)"
+                }
+            }
+        }
+    },
+    "facets": {
+        "status": {
+            "title": "Status"
+        },
+        "award": {
+            "title": "Award"
+        },
+        "lab": {
+            "title": "Lab"
+        },
+        "scope": {
+            "title": "Scope"
+        },
+        "origin": {
+            "title": "Origin"
+        }
+    },
+    "columns": {
+        "accession": {
+            "title": "Accession"
+        },
+        "aliases": {
+            "title": "Aliases"
+        },
+        "status": {
+            "title": "Status"
+        },
+        "lab.title": {
+            "title": "Lab"
+        },
+        "scope": {
+            "title": "Scope"
+        },
+        "origin": {
+            "title": "Origin"
+        },
+        "plasmid_map": {
+            "title": "Plasmid Map"
+        },
+        "quantity": {
+            "title": "Quantity"
+        },
+        "average_size": {
+            "title": "Average Size"
+        },
+        "size_range": {
+            "title": "Size Range"
+        },
+        "crispr_library_detail": {
+            "title": "CRISPR Library Detail"
+        },
+        "mpra_library_detail": {
+            "title": "MPRA Library Detail"
+        }
+    },
+    "boost_values": {
+        "accession": 1.0,
+        "aliases": 1.0,
+        "@type": 1.0,
+        "targeted_gene": 1.0
```

Comment:
As far as I can tell, boost_values for nested properties is not accepted (as it was in encoded). I tried targeted_gene.genes.symbol and other variations but all resulted in errors in cdk-pytest

Reply:
OK